lib/gui/app/components/finish/finish.tsx

```diff
@@ -27,12 +27,9 @@ import { FlashAnother } from '../flash-another/flash-another';
 import { FlashResults } from '../flash-results/flash-results';
 import { SVGIcon } from '../svg-icon/svg-icon';
 
-const restart = (options: any, goToMain: () => void) => {
-	if (!options.preserveImage) {
-		selectionState.deselectImage();
-	}
+function restart(goToMain: () => void) {
 	selectionState.deselectAllDrives();
-	analytics.logEvent('Restart', options);
+	analytics.logEvent('Restart');
```

Comment:
Why remove the options from the log? It was useful to also have the flash information (e.g. the image flashed/reflashed)

Reply:
It was always the same options: `{ preserveImage: true }`

Reply:
I must've confused it with another event